src/modules/account/components/account-deposit/account-deposit.jsx

```diff
@@ -23,6 +30,16 @@ export default class AccountDeposit extends Component {
       height: 'auto',
       width: '100%',
     }
+    const shapeShiftButton = {
+      clear: 'both',
+      float: 'left',
+      fontWeight: 'bold',
```

Comment:
Use less variable here when it is moved over.

Reply:
I'm not really familiar with less. What is the variable I should be using?

Reply:
I think you found it but if not it should be the less file in the same directory.